### PR TITLE
Add URL parameters to read all redirect rules section

### PR DESCRIPTION
**Pull request by jcollado**
This includes both pagination and filtering parameters.

Related: Constructor-io/autocomplete#1204

### CODE REVIEW

File: source/includes/_redirect_rules.md [hunk 1]
```diff
@@ -138,6 +138,18 @@ curl -X GET \
 
 `GET https://ac.cnstrc.com/v1/redirect_rules?key=[your autocomplete key]`
 
+### URL Parameters
+
+Parameter | Required | Description
+--------- | -------- | -----------
+key | Yes | The index you'd like to to retrieve results from.
+num_results_per_page | No | The number of rules to return. Defaults to 20.
+page | No | The page of results to return. Defaults to 1.
```
Comment (adetch) [inline source/includes/_redirect_rules.md]: s/results/redirect rules for consistency.

File: source/includes/_redirect_rules.md [hunk 2]
```diff
@@ -138,6 +138,18 @@ curl -X GET \
 
 `GET https://ac.cnstrc.com/v1/redirect_rules?key=[your autocomplete key]`
 
+### URL Parameters
+
+Parameter | Required | Description
+--------- | -------- | -----------
+key | Yes | The index you'd like to to retrieve results from.
```
Comment (adetch) [inline source/includes/_redirect_rules.md]: The index you'd like to retrieve redirect rules for.

File: source/includes/_redirect_rules.md (hunk 1 above)
Comment (jcollado) [reply source/includes/_redirect_rules.md]: Fixed in 062856d

File: source/includes/_redirect_rules.md (hunk 2 above)
Comment (jcollado) [reply source/includes/_redirect_rules.md]: Fixed in 062856d

**Comment by jcollado** [comment]
Merging as per @adetch comment in Slack

**Pull request merged**